CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Debug)
 endif()
 
+if(NOT LSTORE_PROJECT_VERSION)
+    set(LSTORE_PROJECT_VERSION "0.5.1-unknown_commit")
+endif()
```

Comment:
This can be set like the following below:

```
set(LSTORE_REPO_URL "http://repo.accre.vanderbilt.edu"
         CACHE STRING "URL for lstore-release")
```